src/utils/gauleg.cpp

```diff
@@ -0,0 +1,53 @@
+//========================================================================================
+// Athena++ astrophysical MHD code
+// Copyright(C) 2014 James M. Stone <jmstone@princeton.edu> and other code contributors
+// Licensed under the 3-clause BSD License, see LICENSE file for details
+//========================================================================================
+
+// Athena headers
+#include "../athena.hpp"
+
+
+//======================================================================================
+//! \file Gauleg.cpp
+//======================================================================================
+
+
+
+//--------------------------------------------------------------------------------------
+//! \fn  void Gauleg(int n, Real x1, Real x2,  AthenaArray<Real> &x,
+//          AthenaArray<Real> &w);
+// \brief gauss-legendre weight routine from numerical recipes
```

Comment:
cannot include Numerical Recipes source code in this project

Reply:
This is not direct copy of source code. It is adapted to Athena. Let me check where it is used.

Reply:
> This is not direct copy of source code. It is adapted to Athena. Let me check where it is used.

@felker Please take a look and see if the current files look good.